### PR TITLE
fix qr scanner

### PR DESCRIPTION
**Pull request by thejavascriptman**

### CODE REVIEW

**Comment by A6GibKm** [comment]
This uses the webcam?

File: org.electrum.electrum.json [hunk 1]
```diff
@@ -27,32 +27,28 @@
...
+            "--without-npapi",
+            "--without-xshm",
+            "--with-python=python3",
+            "--disable-doc",
+            "--enable-codes=qrcode"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "https://www.linuxtv.org/downloads/zbar/zbar-0.23.tar.gz",
```
Comment (A6GibKm) [inline org.electrum.electrum.json]: 0.23.1 is available.

**Comment by A6GibKm** [comment]
bot, build org.electrum.electrum

**Comment by thejavascriptman** [comment]
> This uses the webcam?

yes

File: org.electrum.electrum.json (hunk 1 above)
Comment (thejavascriptman) [reply org.electrum.electrum.json]: 0.23.1 fails to build because of some error related to  gettext.h

Comment (A6GibKm) [reply org.electrum.electrum.json]: If I had to guess I would think it is related to gcc10 which is what the latest runtimes use.

Comment (A6GibKm) [reply org.electrum.electrum.json]: https://github.com/mchehab/zbar/issues/90. Just apply the patch found in https://github.com/mchehab/zbar/commit/a133aea7880bbb56d75535e534716d0e16a3b61a.

Comment (A6GibKm) [reply org.electrum.electrum.json]: It might also be a good idea to use the github source.

Comment (thejavascriptman) [reply org.electrum.electrum.json]: updated, git source does not throw a gettex error 🤷‍♂

**Comment by thejavascriptman** [comment]
bot, build org.electrum.electrum

**Comment by A6GibKm** [comment]
@thejavascriptman I want to merge this, but the build fails. I can't locally build electrum currently so it is a little hard for me to fix (Cython or pyqt fails due to memory issues).

**Comment by A6GibKm** [comment]
Actually, just rebase to the latest master.

**Comment by A6GibKm** [comment]
Thanks for your contribution, but please note that the camera is only available in black and white (is this intended?) and works on xorg but not in wayland.

**Pull request merged**